src/main/scala/interplay/PlayRootProjectBase.scala

```diff
@@ -16,9 +16,9 @@ object PlayRootProjectBase extends AutoPlugin {
   override def projectSettings = PlayNoPublishBase.projectSettings ++ Seq(
     crossScalaVersions := {
       if ((ThisBuild / playCrossBuildRootProject).?.value.exists(identity)) {
-        Seq(ScalaVersions.scala212, ScalaVersions.scala213)
```

Comment:
Remove this if-else entirely?

Reply:
And maybe even remove `playCrossBuildRootProject` and let other projects set `crossScalaVersions` explicitly if they need to.

Reply:
> Remove this if-else entirely?

Yes, I was thinking the same yesterday, but wasn't 100% sure. Actually since this is about root projects, which usually don't get published, I think its ok if we remove the default.

> And maybe even remove `playCrossBuildRootProject` and let other projects set `crossScalaVersions` explicitly if they need to.

Done. I set `crossScalaVersions` in the scripted tests explicitely. The projects that I mentioned in this thread will not be affected, because like written they already set the cross version explicitely anyway.